package.json

```diff
@@ -34,6 +35,7 @@
     "node-sass": "^4.12.0",
     "resolve-url-loader": "^3.1.0",
     "sass-loader": "^7.1.0",
+    "vendor-copy": "^2.0.0",
```

Comment:
I don't see it being used anymore, can you remove it?

Reply:
Done.